_includes/header.html

```diff
@@ -1,9 +1,15 @@
 <header>
   <div class="pure-g">
-    <div class="pure-u-1 pure-u-lg-1-5">
-        <a href="{{ site.baseurl }}/">
-          <img class="nav-icon" src={{ site.logo }} alt="MyPaint" />
-        </a>
+    <div class="logo pure-u-1 pure-u-lg-1-5">
+      <a href="{{ site.url }}">
```

Comment:
that was giving me a relative link to whatever page we were on. (e.g when viewed on the "about" page, the link was "/about" isntead of "/"